server/internal/infrastructure/mongo/model_test.go

```diff
@@ -0,0 +1,442 @@
+package mongo
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/reearth/reearth-cms/server/internal/usecase/repo"
+	"github.com/reearth/reearth-cms/server/pkg/id"
+	"github.com/reearth/reearth-cms/server/pkg/key"
+	"github.com/reearth/reearth-cms/server/pkg/model"
+	"github.com/reearth/reearthx/mongox"
+	"github.com/reearth/reearthx/mongox/mongotest"
+	"github.com/reearth/reearthx/rerror"
+	"github.com/reearth/reearthx/usecasex"
+	"github.com/samber/lo"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_ModelRepo_Filtered(t *testing.T) {
```

Comment:
Reimplement it as a simple test, because what this function tests is already tested by other test functions:

```go
assert.Equal(t, &modelRepo{
  filter: repo.ProjectFilter{
    // ...
  }
}, (&modelRepo{
  filter: repo.ProjectFilter{
    // ...
  }
}).Filtered(repo.ProjectFilter{
    // ...
}))
```

Reply:
Done!